integration/spark/src/test/scala/org/apache/carbondata/geo/GeoTest.scala

```diff
@@ -639,6 +639,14 @@ class GeoTest extends QueryTest with BeforeAndAfterAll with BeforeAndAfterEach {
         Row(120197093, 30325985),
         Row(120196020, 30321651),
         Row(120198638, 30323540)))
+    checkAnswer(
```

Comment:
Can do checkanswer with previous query. Can assign previous query to DF

Reply:
done